src/pages/RoleSelection.tsx

```diff
@@ -2,16 +2,21 @@ import { FC } from 'react';
 import { Div1, H1, P, Div2, Div3, Button2 } from './RoleSelection.styles';
 import { useTranslation } from 'react-i18next';
 import { useNavigate } from 'react-router-dom';
-import { useDispatch, useSelector } from 'react-redux';
+import { useSelector } from 'react-redux';
 import { Radio, RadioChangeEvent } from 'antd';
 import { RootState } from 'redux/store';
-import { saveRole } from 'redux/reducers/userSlice';
+import { saveRole, saveUserId } from 'redux/reducers/userSlice';
+import { useAppDispatch } from 'redux/hooks';
+import { useSignUpMutation } from 'service/httpService';
 
 const RoleSelection: FC = () => {
 	const { t } = useTranslation();
 	const navigate = useNavigate();
-	const dispatch = useDispatch();
+	const dispatch = useAppDispatch();
+	const [signUp] = useSignUpMutation();
 	const role = useSelector((state: RootState) => state.user.role);
+	const email = useSelector((state: RootState) => state.user.email);
+	const password = useSelector((state: RootState) => state.user.password);
```

Comment:
why we select 3 values role, email, pass if we just can get user? it make component bigger and I don't really see reason

Reply:
removed